src/sentry/static/sentry/app/views/organizationIntegrations/providerRow.tsx

```diff
@@ -143,6 +157,9 @@ const ProviderDetails = styled(Flex)`
   font-size: 0.8em;
 `;
 
+const StatusContainer = styled(Flex)`
+  align-items: center;
+`;
 type StatusProps = {
```

Comment:
@NisanthanNanthakumar ultra nit: can we add a new line before `type StatusProps`?